firmware/Core/Src/adcs_drivers/adcs_internal_drivers.c

```diff
@@ -167,7 +167,7 @@ uint8_t ADCS_convert_uint32_to_reversed_uint8_array_members(uint8_t *array, uint
     return 0;
 }
 
-/// @brief Convert a double into a string with a given decimal precision.
```

Comment:
The brief should describe what the function does, not why it does it. Consider adding this note in a `/// @note` tag instead.

Reply:
Done! Thanks.